cmd/client/lib/cli.go

```diff
@@ -95,21 +102,30 @@ var ClientFlags = []cli.Flag{
 func Create(c *cli.Context, withInstrumentation bool, opts ...client.Option) (client.Client, error) {
 	clients := make([]client.Client, 0)
 	var info *chain.Info
+	var err error
+
+	if c.IsSet(GroupConfFlag.Name) {
+		info, err = chainInfoFromGroupTOML(c.Path(GroupConfFlag.Name))
```

Comment:
worth checking if `chain-hash` conflicts with the loaded group?

Reply:
I wouldn't expect users to typically provide _both_ `-hash` and `-group-conf` params. If we provide `-group-info` then it'll use `http.NewWithInfo` which will ignore the `WithChainHash` option that `-hash` will set. I guess that's a good enough reason to just make sure they're the same.